dvd/dvd_dump.ixx

```diff
@@ -525,22 +648,23 @@ export bool redumper_dump_dvd(Context &ctx, const Options &options, DumpMode dum
 
                     auto layer_descriptor = (READ_DVD_STRUCTURE_LayerDescriptor &)structure[sizeof(CMD_ParameterListHeader)];
 
+                    uint32_t layer_break = 0;
+
                     // opposite
                     if(layer_descriptor.track_path)
                     {
-                        int32_t lba_first = sign_extend<24>(endian_swap(layer_descriptor.data_start_sector));
-                        initial_layer0_last = sign_extend<24>(endian_swap(layer_descriptor.layer0_end_sector));
+                        uint32_t lba_first = sign_extend<24>(endian_swap(layer_descriptor.data_start_sector));
+                        uint32_t layer0_last = sign_extend<24>(endian_swap(layer_descriptor.layer0_end_sector));
```

Comment:
These were originally `int32_t` in redumper, was there a particular reason to convert to the unsigned version?